packages/api/src/routers/api/team.ts

```diff
@@ -158,4 +166,38 @@ router.get('/tags', async (req, res, next) => {
   }
 });
 
+router.delete(
+  '/users/:id',
+  validateRequest({
+    params: z.object({
+      id: z.string(),
+    }),
+  }),
+  async (req, res, next) => {
+    try {
+      const id = req.params.id;
+      const user = await findUserById(id);
+      const teamInvite = await TeamInvite.findOne({
```

Comment:
imo feels weird to overload the endpoint to delete two different resources that have no overlap. Let's just keep this endpoint to deleting users for now, we can create a separate one for deleting invites.

Reply:
changes made.